src/pages/AccountPage/index.tsx

```diff
@@ -1,38 +1,63 @@
+import { useState, useEffect } from 'react';
+import Cookies from 'js-cookie';
 import { Box, Grid, Typography } from '@mui/material';
 import { AccountInput } from '../../components';
 import THEME from '../../theme';
 import { WrappBtn } from '../../components/AccountInput/acount.styled';
 
-const AccountPage = () => (
-  <Box>
-    <Typography color={THEME.palette.custom.white} sx={{ fontWeight: 'bold', fontSize: '2vw', marginBottom: '2vw' }}>
-      Edit Profile Information
-    </Typography>
-    <Grid container spacing={3} sx={{ marginBottom: '2vw' }}>
-      <Grid item xs={6}>
-        <AccountInput label="Name" />
-      </Grid>
-      <Grid item xs={6}>
-        <AccountInput label="Email" />
-      </Grid>
-      <Grid item xs={6}>
-        <AccountInput label="Old Password" />
-      </Grid>
-      <Grid item xs={6}>
-        <AccountInput label="New Password" />
-      </Grid>
-    </Grid>
+interface AccountData {
+  name: string,
+  email: string,
+  phone: string,
+}
+
+const AccountPage = () => {
+  const [name, setUsername] = useState<string>('');
+  const [email, setEmail] = useState<string>('');
+  const [phone, setPhone] = useState<string>('');
+
+  useEffect(() => {
+    const token = Cookies.get('token');
+    if (token) {
+      // eslint-disable-next-line no-shadow
+      const { name, email, phone } = JSON.parse(atob(token)) as AccountData;
+      setUsername(name);
+      setEmail(email);
+      setPhone(phone);
+    }
+  }, []);
+  return (
     <Box>
-      <Typography color={THEME.palette.custom.white}>
-        <Typography sx={{ marginBottom: '1vw' }}>
-          Delete Your Account
-        </Typography>
-        <WrappBtn>
-          Delete Account
-        </WrappBtn>
+      <Typography color={THEME.palette.custom.white} sx={{ fontWeight: 'bold', fontSize: '2vw', marginBottom: '2vw' }}>

```

Comment:
```suggestion
      <Typography sx={{ color: 'custom.white'  fontWeight: 'bold', fontSize: '2vw', marginBottom: '2vw' }}>

```
 you can directly use the color value from the theme without referencing the entire path.

Reply:
Thanks for your technical notes 👌👏
Happy Eid🌹